website/src/app/kb/client-apps/windows-headless-client/readme.mdx

```diff
@@ -0,0 +1,224 @@
+with a few environment variables like so:
+
+```pwsh
+> $env:FIREZONE_TOKEN="YOUR_TOKEN_HERE"
+> $env:RUST_LOG="info"
+> .\firezone-client-headless-windows_<VERSION>_x86_64.exe
+```
+
+By default, the client only logs to `stdout` on level `error` and above. Set
+`RUST_LOG="info"` to see more logs, and `LOG_DIR` to write logs to disk. For a
+full list of environment variables, see
+[below](#environment-variable-reference).
+
+Most environment variables can also be set in the command line. For a full list,
+see [help output](#help-output).
+
+### Disabling split DNS
+
+By default, Split DNS is **enabled** for the Windows Headless Client. In most
+cases, this is what you want.
+
+If you're experiencing DNS issues or incompatibilities with other DNS software
+on your system, and you don't need to access DNS Resources, you can disable
+Split DNS.
+
+To do this, set the `FIREZONE_DNS_CONTROL` environment variable to `disabled`.
+
+[Read more](/kb/deploy/dns) about how DNS works in Firezone.
+
+### Environment variable reference
+
+| Variable Name          | Default Value       | Description                                                                                                                                                                   |
+| ---------------------- | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `FIREZONE_TOKEN`       |                     | Service account token generated by the portal to authenticate this Client.                                                                                                    |
+| `FIREZONE_NAME`        | `<system hostname>` | Friendly name for this client to display in the UI.                                                                                                                           |
+| `FIREZONE_ID`          |                     | Identifier used by the portal to unqiuely identify this client.                                                                                                               |
+| `FIREZONE_DNS_CONTROL` | (blank)             | The DNS control method to use. The default is `nrpt`, the only supported option on Windows. Set this to `disabled` to disable DNS control to route IP or CIDR resources only. |
+| `LOG_DIR`              |                     | File logging directory. Should be a path that's writeable by the current user. If unset, logs will be written to `stdout` only.                                               |
+| `RUST_LOG`             | `error`             | Log level for the client. Set to `debug` for verbose logging. Read more about configuring Rust log levels [here](https://docs.rs/env_logger/latest/env_logger/).              |
+
+### Help output
+
+```text
+PS C:\Users\jamil\Downloads> .\firezone-client-headless-windows_1.4.2_x86_64.exe help
```

Comment:
```suggestion
PS C:\Users\jamil\Downloads> .\firezone-client-headless-windows_1.4.2_x86_64.exe --help
```

?

Reply:
It says in the output below that `help` is a subcommand?

Reply:
Seems like they both emit the same output, but updated it to --help anyway.